utils/mime.go

```diff
@@ -40,6 +40,9 @@ const (
 
 // WithLazyMIMEType attaches the lazy suffix to a MIME.
 func WithLazyMIMEType(mimeType string) string {
+	if _, has := CheckLazyMIMEType(mimeType); has {
+		return mimeType
+	}
```

Comment:
Assuming we never want to have multiple `+lazy` appended to MIME types. Seemed safer to do this then have the caller  check if `+lazy` is present before calling this function.

Reply:
yes! I was just gonna suggest this in server.go, good catch.